pkg/controllers/nodeclaim/tagging/controller.go

```diff
@@ -95,8 +99,9 @@ func (c *Controller) Register(_ context.Context, m manager.Manager) error {
 
 func (c *Controller) tagInstance(ctx context.Context, nc *karpv1.NodeClaim, id string) error {
 	tags := map[string]string{
-		v1.TagName:      nc.Status.NodeName,
-		v1.TagNodeClaim: nc.Name,
+		v1.TagName:              nc.Status.NodeName,
+		v1.TagNodeClaim:         nc.Name,
+		v1.EKSClusterNameTagKey: options.FromContext(ctx).ClusterName,
```

Comment:
Do we pass the cluster name into this context? I know we do normally, but want to make sure it's at the right stage where the settings are already parsed in

Reply:
> stage where the settings are already parsed in

Yep, they're here